docs/adr/0005-use-github-actions-for-vsp-platform-ci.md

```diff
@@ -1,40 +1,50 @@
-# Context
+# 5. Use GitHub Actions For VSP Platform CI
 
-https://dsva.slack.com/archives/C01CJV0L9PS/p1612651359029600?thread_ts=1612651359.029600&cid=C01CJV0L9PS
+Date: 2021-02-16
 
-TL;DR: CircleCI as implemented with VA restrictions does not allow easy management of permissions on ‘Contexts’ which contain credentials which are available to the entire ‘department-of-veterans-affairs’ Github Org. Although they are not displayed in plaintext in the CircleCI UI, they are obtainable in plaintext by running a build in CircleCI and using SSH to connect to your in-progress build, and outputting ENV variables. If the entire Org has access to the context, there is no way to stop anyone in the Org from doing so.
+Whom:
+- Demian Ginther
+- Jesse House
+- Michael Fleet
+- Dror Matalon
+## Status
+
+Accepted
+
+## Context
+
+[Slack discussion thread](https://dsva.slack.com/archives/C01CJV0L9PS/p1612651359029600?thread_ts=1612651359.029600&cid=C01CJV0L9PS).
+
+### CircleCI + VA Security
+
+CircleCI as implemented with VA restrictions does not allow easy management of permissions on ‘Contexts’ which contain credentials which are available to the entire ‘department-of-veterans-affairs’ Github Org. Although they are not displayed in plaintext in the CircleCI UI, they are obtainable in plaintext by running a build in CircleCI and using SSH to connect to your in-progress build, and outputting ENV variables. If the entire Org has access to the context, there is no way to stop anyone in the Org from doing so.
 
 In addition, CircleCI as implemented with VA restrictions does not allow use of ‘third-party’ Orbs (reusable code) which means that we cannot publish our own Orbs either.
```

Comment:
I *believe* that the VA could decide to allow 3rd party orbs. It's still not clear to me who is blocking this.

Reply:
I agree that they _could_ decide to do this, but I am not sure that they will at this point, and the functionality we need is already included in Github Actions re: creating reusable workflows/actions